alpine/latest/Dockerfile

```diff
@@ -0,0 +1,91 @@
+###############################################################
+#
+# Copyright (C) 2019 James Fuller <jim.fuller@webcomposite.com>
+#
+# SPDX-License-Identifier: MIT
+#
+# pinning to Alpine 3.10.2 as base image
+###############################################################
+FROM registry.hub.docker.com/library/alpine:3.10.2 AS builder
+
+###############################################################
+# set build args
+###############################################################
+ARG CURL_RELEASE_TAG=latest
+ARG CURL_GIT_REPO=https://github.com/curl/curl.git
+ARG CURL_CONFIGURE_OPTION
+ARG LABEL_VERSION=1.0.0
+ARG LABEL_NAME=curl
+ARG LABEL_DESC=curl
+
+###############################################################
+# build curl
+###############################################################
+# install deps and use latest curl release source
+RUN \
+    apk add --no-cache \
+       autoconf    \
+       automake    \
+       build-base  \
+       curl-dev    \
+       groff       \
+       libtool     \
+    && \
+    rm -rf "/var/cache/apk/"*
+
+RUN mkdir /src
+COPY "curl"  "/src/curl"
+WORKDIR "/src/curl"
+
+RUN ./buildconf && \
+    autoreconf -vif && \
+    ./configure ${CURL_CONFIGURE_OPTION} &&\
+    make -j$(nproc) &&\
+    make DESTDIR="/alpine/" install  -j$(nproc)
+
+###############################################################
+# pinning to Alpine 3.10.2
+###############################################################
+FROM registry.hub.docker.com/library/alpine:3.10.2
+
+ARG CURL_RELEASE_TAG=latest
+ARG CURL_RELEASE_VERSION
+ARG CURL_GIT_REPO=https://github.com/curl/curl.git
+
+ENV CURL_VERSION ${CURL_RELEASE_VERSION}
+ENV CURL_RELEASE_TAG ${CURL_RELEASE_TAG}
+ENV CURL_GIT_REPO ${CURL_GIT_REPO}
+
+###############################################################
+# define docker labels
+###############################################################
+LABEL Maintainer="James Fuller <jim.fuller@webcomposite.com>"
+LABEL Name="curl"
+LABEL Version="${LABEL_VERSION}"
+LABEL docker.cmd="docker run -it curl/curl:${CURL_RELEASE_VERSION} http://curl.haxx.se"
+
+###############################################################
+# remove curl
+###############################################################
+RUN apk del --no-cache curl && \
```

Comment:
Is `curl` present in a default alpine container?

Reply:
right you are - curl does not exist in alpine base image .. no need for this nonsense